pkgs/servers/mfi/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "mfi-controller-${version}";
+  version = "2.1.11";
+
+  src = fetchurl {
+    url = "https://dl.ubnt.com/mfi/${version}/mFi.unix.zip";
+    sha256 = "0b9q6025zf9zjzq8dcmcyai8rslx67g52j41gacxsk9i5dspmw90";
+  };
+
+  buildInputs = [ unzip ];
+
+  phases = [ "unpackPhase" "installPhase" ];
```

Comment:
If you're only trying to avoid the build phase, it may be beneficial to disable it specifically with `dontBuild = true`